From ad9ab3b3c553cbc3c61f233b6e2cd5abdd2a624b Mon Sep 17 00:00:00 2001
From: Otavio Salvador <otavio@ossystems.com.br>
Date: Tue, 23 May 2023 13:16:05 -0300
Subject: [PATCH] ARM: imx_v7_defconfig: Remove KERNEL_LZO config

The KERNEL_GZIP is used in most config and is the default, there is no
clear reason to diverge so let default be used.

Upstream-Status: Pending

Signed-off-by: Otavio Salvador <otavio@ossystems.com.br>
---
 arch/arm/configs/imx_v7_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/configs/imx_v7_defconfig b/arch/arm/configs/imx_v7_defconfig
index 6de3049689191..94e9457e2bab1 100644
--- a/arch/arm/configs/imx_v7_defconfig
+++ b/arch/arm/configs/imx_v7_defconfig
@@ -1,4 +1,3 @@
-CONFIG_KERNEL_LZO=y
 CONFIG_SYSVIPC=y
 CONFIG_POSIX_MQUEUE=y
 CONFIG_NO_HZ=y
-- 
2.40.1

